packages/fortune.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Fortune < Package
+  description 'Fortune is a small game that is meant to lighten up your life. It can be used to display a random entry from a cookie file.'
+  homepage 'https://www.fefe.de/fortune/'
+  version '1.2'
+  source_url 'http://dl.fefe.de/fortune-1.2.tar.bz2'
+  source_sha256 'cbb246a500366db39ce035632eb4954e09f1e03b28f2c4688864bfa8661b236a'
+
+  depends_on "fortunes"
+
+  def self.build
+    text = File.read("fortune.c")
+    new_contents = text.gsub(/\/usr/, "#{CREW_PREFIX}")
+    File.open("fortune.c.t", "w") {|file| file.puts new_contents }
+    system "mv fortune.c.t fortune.c"
+    system "make"
+  end
+
+  def self.install
+    system "mkdir -pv #{CREW_DEST_PREFIX}/bin"
+    system "mv fortune #{CREW_DEST_PREFIX}/bin/"
```

Comment:
You could replace these two lines with one:
```
system "install -Dm755 fortune #{CREW_DEST_PREFIX}/bin/"
```

Reply:
Did that for `sl` as well :)